spec/unit/views/components/paginated_collection_spec.rb

```diff
@@ -200,7 +200,7 @@ def paginated_collection(*args)
       end
 
       it "should show the proper item counts" do
-        expect(pagination.find_by_class('pagination_information').first.content.gsub('&nbsp;',' ')).
+        expect(pagination.find_by_class('pagination-information').first.content.gsub('&nbsp;',' ')).
```

Comment:
Line is too long. [100/80]<br>Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.